mcd/mcd-seth/mcd-seth.md

```diff
@@ -133,184 +170,243 @@ In order to better understand the MCD contracts, the following provides a brief
 - `lad`: Vault owner
 - `rat`: collateralization ratio
 
-After giving permission to the BAT adapter of MCD to take some of our tokens, it’s time to finally start using the MCD contracts.  
+After giving permission to the WBTC adapter of MCD to take some of our tokens, it’s time to finally start using the MCD contracts.
+
 We'll be using the [CDP Manager](https://github.com/makerdao/dss-cdp-manager) as the preferred interface to interact with MCD contracts.
 
-We begin by opening an empty Vault so we can use it to lock collateral into. For this we need to define the type of collateral (BAT-A) we want to lock in this Vault.  
-`export ilk=$(seth --to-bytes32 $(seth --from-ascii "BAT-A"))`  
-Now let's open the Vault  
-`seth send $CDP_MANAGER 'open(bytes32,address)' $ilk $ETH_FROM`
+We begin by opening an empty Vault, so we can use it to lock collateral into. For this we need to define the type of collateral (WBTC-A) we want to lock in this Vault:
+```bash
+export ilk=$(seth --to-bytes32 $(seth --from-ascii "WBTC-A"))
+```
 
-We need the `cdpId` and `urn` address of our open Vault so we can interact with the system.
+Now let’s open the Vault:
+```bash
+seth send $CDP_MANAGER 'open(bytes32, address)' $ilk $ETH_FROM
+```
+
+We need the `cdpId` and `urn` address of our open Vault, so we can interact with the system.
 
 ```bash
-export cdpId=$(seth --to-dec $(seth call $CDP_MANAGER 'last(address)' $ETH_FROM))
-export urn=$(seth call $CDP_MANAGER 'urns(uint)(address)' $cdpId)
+export cdpId=$(seth call $CDP_MANAGER 'last(address)(uint256)' $ETH_FROM)
+export urn=$(seth call $CDP_MANAGER 'urns(uint256)(address)' $cdpId)
 ```
 
 After acquiring `cdpId` and `urn` address, we can move to the next step. Locking our tokens into the system.
-First we are going to make a transaction to the BAT adapter to actually take 10 of our tokens with the join contract function.
 
-The contract function looks like the following: `join(address urn, uint wad)`
+First we are going to make a transaction to the WBTC adapter to actually take 5 of our tokens with the join contract function.
+The contract function looks like the following: `join(address urn, uint256 wad)`
```

Comment:
I'd suggest to maybe add a note to specify that the WBTC adapter uses a non standard GemJoin ([GemJoin5](https://github.com/makerdao/dss-gem-joins/blob/master/src/join-5.sol)).
```suggestion
The contract function looks like the following: `join(address urn, uint256 amt)`
```